src/tespy/networks/networks.py

```diff
@@ -2511,6 +2511,134 @@ def postprocessing(self):
+            Energy flows in network. Used to calculate product exergy
+            or fuel exergy of turbines and pumps.
+        E_F  : list
+            List containing components which represent fuel exergy input
+            in network.
+        E_L  : list
+            List containing connections which represent exergy loss streams.
+
+        Note
+        ----
+            .. math::
+
+                E_{\text{D},comp} = E_{\text{F},comp} - E_{\text{P},comp}\\
+                \varepsilon_{\text{comp}} =
+                \frac{E_{\text{P},comp}}{E_{\text{F},comp}}
+
+                E_{\text{D,tot,bottom up}} = \sum_{comp=0}^N E_{\text{D},comp}
+
+                E_{\text{P}} = P_{\text{net}}\\
+                E_{\text{F}} = \sum_{comp=0}^N E_{\text{F},comp}\\
+                E_{\text{L}} = \sum_{conn=0}^N E_{\text{PH},conn}\\
+                E_{\text{D}} = E_{\text{F}} - E_{\text{P}} - E_{\text{L}}\\
+                \varepsilon = \frac{E_{\text{P}}}{E_{\text{F}}}
+
+                y_{\text{D},comp} =
+                \frac{\dot{E}_{\text{D},comp}}{\dot{E}_{\text{F}}}\\
+                y^*_{\text{D},comp} =
+                \frac{\dot{E}_{\text{D},comp}}{\dot{E}_{\text{D}}}
+        """
+        if E_F is None:
+            msg = ('Missing fuel exergy E_F of network.')
+            logging.warning(msg)
+
+        # physical exergy of connections
+        for conn in self.conns.index:
+            conn.get_physical_exergy(pamb, Tamb)
+
+        # exergy balance of components
+        for cp in self.comps.index:
+            if (isinstance(cp, cycle_closer) or isinstance(cp, sink) or
+                    isinstance(cp, source) or isinstance(cp, splitter)):
+                continue
+            cp.exergy_balance(Tamb, bus)
+
+            # calculate E_D for components
+            try:
+                if isinstance(cp.E_P, str):
+                    cp.E_D = cp.E_F
+                else:
+                    cp.E_D = cp.E_F - cp.E_P
+            except AttributeError:
+                cp.E_D = 0
+                cp.E_F = 0
+                cp.E_P = 0
+            # calculate epsilon for components
+            try:
+                if isinstance(cp.E_P, str):
+                    cp.epsilon = 'n/a'
+                else:
+                    cp.epsilon = cp.E_P / cp.E_F
+            except ZeroDivisionError:
+                cp.epsilon = -111
+
+        # bottom up calculation of exergy destruction
+        self.E_D_sum = 0
+        for cp in self.comps.index:
+            if (isinstance(cp, cycle_closer) or isinstance(cp, sink) or
+                    isinstance(cp, source) or isinstance(cp, splitter)):
+                continue
+            if cp.epsilon == -111:
+                continue
+            self.E_D_sum += cp.E_D
+
+        # calculate E_P, E_F, E_L, E_D and epsilon for network
+        self.E_L = 0
+        self.E_F = 0
+
+        for b in self.busses.values():
+            self.E_P = b.P.val*-1
```

Comment:
Some busses may not be used for exergy balancing. E.g. a bus can be used, to relate the power values of two components but is not connected to any physical out- or input of energy of the network. A solution could be, that the user needs to pass the busses, that should be considered in this method.

Reply:
Good point. 
How can I access the net power of a bus? Is it the same way I did it here?